### ISSUE TITLE
OS X graphical issues

### ISSUE THREAD

**Issue by chief1983** [scp-fs2open/fs2open.github.com#969]
[Youtube demo](https://youtu.be/XVFI3ULIibg)

[fs2_open.log](https://github.com/scp-fs2open/fs2open.github.com/files/467944/fs2_open.txt)

There are several things that are really wrong looking on my current setup.  From the log, I should have the OpenGL version high enough, and we are now on the core profile, so I'd hoped things would be better.  Still having some issues though.  @SamuelCho, the ability to run OS X builds now should be greatly improved.

I attempted to run the game with -fb_thrusters as well, but wxLauncher doesn't seem to be acknowledging my changing the settings.  Won't acknowledge a resolution change either.

Also, I ran the video recording with a release build, the log is from a follow up run with a fast debug build.

**Comment by SamuelCho** [comment]
Glad to finally be able to debug graphics on Mac. Is this in master yet or is it on one of your branches?

**Comment by chief1983** [comment]
It was merged over the weekend.  Just verified I have the problem with the thrusters with fb thrusters on as well, manually editing my cmdline config file until wxlauncher 0.12.0 is available.

**Comment by SamuelCho** [comment]
Got it, will try to figure out what's up.

**Comment by SamuelCho** [comment]
So initial testing tells me that this has something to do with the g-buffers. Deferred lighting doesn't work on Mac but disabling it should look okay. I'd say just disable deferred lighting as a temporary workaround for now until I can find a solution. Thanks again for unblocking me chief.

**Comment by chief1983** [comment]
Ok yeah, deferred is still an issue.  However, even with it off, the lighting reflections still seem off.  Might just be a shortcoming of their use with the retail models, but maybe not.  The beams on massive battle two will glow entire sides of the Orion and Hecate.  The subachs cause odd glows on the Shivan capital ships as well.  Whole panels blink purple, etc.  Trying to describe in enough detail that I don't need to bother with uploading another video, can't figure out taking a screenshot of the game in OS X at the moment.
